app/src/main/res/values-de/strings.xml

```diff
@@ -68,10 +68,10 @@
     <string name="wind_direction_south_east">SO</string>
     <string name="wind_direction_south_west">SW</string>
     <string name="wind_direction_west">W</string>
-    <string name="label_activity_graphs">Grafiken Wetter</string>
+    <string name="label_activity_graphs">Wettergrafiken</string>
     <string name="label_activity_main">Aktuelle Wetterlage</string>
-    <string name="label_activity_weather_forecast">Daily Forecast</string>
-    <string name="nav_menu_graphs">Graphs</string>
+    <string name="label_activity_weather_forecast">Tägliche Vorhersage</string>
+    <string name="nav_menu_graphs">Grakiken</string>
```

Comment:
Thank you for the hint @firebirdberlin .
That was an early mistake I made. Please see the final version of my changes (l.30 in the German string.xml). There I translated the navigation menu entry <string name="nav_menu_graphs">Diagramme</string> as 'Diagramme', because that would be a common way to name it in German, instead of 'Graphen'.